tests/integration/inhibit-if-kmods-is-not-supported/main.fmf

```diff
@@ -0,0 +1,9 @@
+summary: kmods tests
+# TODO Add prepare with running ansible playbooks
+# TODO why prepare step is ignored?
+prepare:
+  how: shell
+  script: |
+    echo "some preparation script here"
```

Comment:
@psss this step is ignored (when do tmt run). Should I create a separate fmf for running test? i.e. this one remains + add run_tests.fmf with test: pytest?

Reply:
The `prepare` step can be configured under a plan. To better understand the difference between plan (L2) and test (L1) have a look at the [spec overview](https://tmt.readthedocs.io/en/latest/spec.html).

Reply:
OK, thx @psss. I created a ticket to track this https://github.com/psss/tmt/issues/603

Reply:
OK, my plan was to have a possibility to add playbooks for each integration test. So in this case potentially each test should have own plan, right @psss ?